src/cgame/cg_draw.cpp

```diff
@@ -5208,8 +5208,10 @@ static void CG_DrawPlayerStatus(void)
 	if (etj_HUD_weaponIcon.integer)
 	{
 		// Draw ammo
-		// weap = CG_PlayerAmmoValue(&value, &value2, &value3);
 		value = value2 = value3 = 0;
+		// weap = 
+		CG_PlayerAmmoValue(&value, &value2, &value3);
```

Comment:
All of these were previously commented out but now uncommented. Is this intentional or am I missing something?

Reply:
I had commented them out accidentally because of unused return value, should be uncommented